packages/data-plugins/lib/collections/core.ts

```diff
@@ -93,9 +93,22 @@ export class PluginCore extends Plugin {
         stac_extensions: {
           type: 'array',
           label: 'STAC Extensions',
+          'ui:widget': 'tagger',
           items: {
-            label: 'Extension',
-            type: 'string'
+            type: 'string',
+            allowOther: {
+              type: 'string'
+            },
+            enum: [
```

Comment:
When an operator registers a plugin that provides a new extension, they also need to add the extension URL to this list for it to come up in the suggestions.

Should we include that as a note in the plugin dev docs?

Reply:
In theory yes, but it is unlikely that the operators can change the core plugins.
I have an idea on how to handle this, let's discuss.